README.md

```diff
@@ -28,30 +29,41 @@ To install vCluster Rancher Operator, you can add the loft repository to Rancher
 7. Follow the installation process and install the chart.
 8. In the sidebar, navigate to "Workloads" -> "Deployments". Confirm that the deployment named "vcluster-rancher-op" has the State "Active".
 
-Once the operator is installed, all vClusters deployed in any downstream cluster in rancher will cause a corresponding Rancher cluster to be created, the vCluster to connect to the corresponding Rancher cluster, and cluster owners added.
+Once the operator is installed:
+* All vClusters deployed in any downstream cluster will cause a corresponding Rancher cluster to be created
+* The vCluster will connect to the corresponding Rancher cluster
+* Any project-member or above will be added as a cluster-owner
```

Comment:
We should probably explicitly say which roles since we are going off role templates and not actual permissions. Atm its project member and project owners. I hope to have cluster owners and members added as well.